frontend-ssr/components/cli_git/GitCli.module.scss

```diff
@@ -0,0 +1,72 @@
+@import '../../styles/variables';
+
+.commands {
+    width: 35vw;
+
+    .commands__cli {
+        height: 50vh;
+        overflow: auto;
+        padding: 2vh 1vw;
+        @extend %gradientBackground, %blackGradientBackground;
+
+        .command {
+            color:color(blue);
+        }
+        .success {
+            color:color(green);
+        }
+        .error {
+            color: color(red);
+        }
+        p {
+            margin: 0;
+        }
+    }
+}
+.form {
+    position: relative;
+    color: color(white);
+    .form__input {
+        width: 100%;
+        background-color: color(gray);
+        padding: 1em;
+        border: none;
+        &:focus ~ span::after {
+            width: 0;
+        }
+    }
+    span {
```

Comment:
I was confused who this `span` is at first. Let's add a class to it, so we know it's the blinking cursor.